.github/workflows/release.yml

```diff
@@ -138,3 +138,18 @@ jobs:
         tag: ${{ github.ref }}
         file: /tmp/anycloud-macos.tar.gz
         asset_name: anycloud-macos.tar.gz
+
+    - name: Update homebrew
+      # TODO: should we create a new token or reuse the AUTOPUBLISH one?
```

Comment:
Reusing the autopublish token should be fine, I think.

Reply:
Ok, will remove the TODO